### ISSUE TITLE
Update screenshots

### ISSUE THREAD

**Issue by jules32** [Openscapes/quarto-website-tutorial#3]
- [ ] explore.qmd screenshots, for example the screenshot of files on github, after the filetypes stabilize

**Comment by stefaniebutland** [comment]
in https://openscapes.github.io/quarto-website-tutorial/explore.html#the-websites-repo, text is up to date "click inside “quarto-workflows”" but screenshot shows "basic-workflows"

**Comment by jules32** [comment]
Yes this is a perfect example of a screenshot to update! And also the _yml file screenshot

https://openscapes.github.io/quarto-website-tutorial/explore.html#quarto.yml-intro
